subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/InHouseInstrumentationTestRunner.kt

```diff
@@ -280,13 +281,12 @@ abstract class InHouseInstrumentationTestRunner :
         }
     }
 
-    @SuppressLint("LogNotTimber")
-    fun tryToReportUnexpectedIncident(incident: Throwable, tag: String) {
+    fun tryToReportUnexpectedIncident(incident: Throwable) {
         try {
             report.registerIncident(AppCrashException(incident))
             report.reportTestCase()
         } catch (t: Throwable) {
-            Log.w(tag, "Can't register and report unexpected incident", t)
+            logger.critical("Can't register and report unexpected incident", t)
```

Comment:
Is it a failure for tests running?

Reply:
i reverted to warning, first look was: yes, it's completely critical, but then i remember all this multithread calls to onException, and that we will get "false-criticals" here